### ISSUE TITLE
ERROR: [SHUFFLE #2] uninferred dimensions are not an exact divisor of input dimensions, so inferred dimension cannot be calculated

### ISSUE THREAD

**Issue by heaseoEspreso** [NVIDIA-AI-IOT/torch2trt#472]
I was successful to run Alexnet and resnet50. 
However, when I am trying to convert tecoGAN to tensorRT, i am getting issue below

==========================================================

Warning: Encountered known unsupported method torch.nn.functional.interpolate
Warning: Encountered known unsupported method torch.nn.functional.interpolate
Warning: Encountered known unsupported method torch.nn.functional.interpolate
Warning: Encountered known unsupported method torch.Tensor.repeat
Warning: Encountered known unsupported method torch.nn.functional.conv2d
Warning: Encountered known unsupported method torch.Tensor.repeat
Warning: Encountered known unsupported method torch.nn.functional.conv2d
[TensorRT] ERROR: [SHUFFLE #2] torch.Tensor.view(tensor(shape=[8, 4], dtype=torch.float32), -1, 1, 4, 1): uninferred dimensions are not an exact divisor of input dimensions, so inferred dimension cannot be calculated

==========================================================
